### DISCUSSION TITLE
Blueprint .yaml - author: Home Assistant Property author not allowed

### DISCUSSION THREAD

**Discussion by RobOrch** [coder/code-server#6063] [Q&A]
Hi, I have recently installed Home Assistant on a VM and have an error in two blueprints:
notify_leaving_zone.yaml
confirmable_notification.yaml
both have the error: author property not allowed.

In both cases this occurs near the beginning:
source_url: https://github.com/home-assistant/core/blob/dev/homeassistant/components/automation/blueprints/notify_leaving_zone.yaml
  author: Home Assistant

What is the edit to remove the error?
  
Thanks Rob

**Comment by code-asher** [comment]
Was this meant to be posted on https://github.com/home-assistant/core?  This does not seem to be related to code-server.

**Comment by RobOrch** [comment]
Asher,
Probably not - I was having problems (which I have now sorted) and the software said to raise an issue. Not having done this before I probably got it wrong!
You can remove the post.
Thanks,
Rob.


-----Original Message-----
